digital-signage-app-v2/app/components/video-player.js

```diff
@@ -30,13 +30,16 @@ Callbacks:
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-	classNames: ["video-player"],
```

Comment:
this didn't do anything so i removed it. class is applied in the corresponding hbs file